pkgs/development/compilers/elm/packages/node-composition.nix

```diff
@@ -5,8 +5,8 @@
   }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-12_x"}:
 
 let
-  nodeEnv = import ../../../node-packages/node-env.nix {
-    inherit (pkgs) stdenv python2 util-linux runCommand writeTextFile;
+  nodeEnv = import ./node-env.nix {
+    inherit (pkgs) stdenv python2 utillinux runCommand writeTextFile;
```

Comment:
```suggestion
    inherit (pkgs) stdenv python2 util-linux runCommand writeTextFile;
```

Reply:
This file is generated by node2nix. I think this change comes from the new version. If we would revert this, we should also ditch `node-env.nix` I think.

Reply:
@domenkozar what do you think about this? Should I edit generated files?

Reply:
so i this PR I did not change any generated file. This suggestion is a change in file which was generated. This change in how node2nix generates file is also why we now need our own node-env.nix. I'm not sure if anything needs fixing because the newly generated files do work just fine. I think all we would need to avoid creating node-env.nix is would be to regenerate nodePackages as well Which I don't want to do to keep scope of this change small.

Furthermore I think it'll be better for node2nix maintainers if we would break the dependecy on their node-env.nix for a moment so they can do the transition to new format on their own terms without breaking elmPackages.